docs/messenger/api.md

```diff
@@ -239,6 +239,8 @@
 | conversation | [Conversation](#berty.messenger.v1.Conversation) |  |  |
 | state | [Contact.State](#berty.messenger.v1.Contact.State) |  |  |
 | display_name | [string](#string) |  |  |
+| created_date | [int64](#int64) |  |  |
+| sent_date | [int64](#int64) |  | specific to outgoing requests |
```

Comment:
What is the difference beetween `created_date` and `sent_date` ? Is there a case where we create a request and forward it later ?

Reply:
In the outgoing request case, if the contact we want to request is not online, the contact will stay in `OutgoingRequestEnqueued` state until the contact receives the request.
`created_date` is the date at which we received the `OutgoingRequestEnqueued` or `IncomingRequest` event.
`sent_date` is the date at which we received the `OutgoingRequestSent` event.

Reply:
:ok_hand: